trigger/kafkasub/README.md

```diff
@@ -0,0 +1,128 @@
+# tibco-kafkasub
+This trigger provides your flogo application with the ability to subscribe to messages from a kafka cluster and start a flow with the contents of the message.  It is assumed that the messages plain text.  The trigger supports TLS and SASL.  
+To make a TLS connection specifiy a trust dir containing the caroots for your kafka server and a broker URL which points to an SSL port.
+To use SASL simply provide the username and password in the settings config.
+
+
+## Installation
+
+```bash
+flogo add trigger github.com/TIBCOSoftware/flogo-contrib/trigger/kafkasub
```

Comment:
Should this be?
flogo install github.com/TIBCOSoftware/flogo-contrib/trigger/kafkasub